doc/conf.py

```diff
@@ -253,7 +253,7 @@
 # key: first image in set
 # values: (number of plot in set, height of thumbnail)
 carousel_thumbs = {'sphx_glr_plot_classifier_comparison_001.png': 600,
-                   'sphx_glr_plot_outlier_detection_003.png': 372,
+                   'sphx_glr_plot_lof_novelty_detection_001.png': 372,
```

Comment:
the example to use is `plot_anomaly_comparison`, not `plot_lof_novelty_detection`.

Reply:
Yes, I change the example case from novelty detection to anomaly comparison.
Thanks. Happy Christmas.